frontend/src/individual_add.tsx

```diff
@@ -328,6 +338,10 @@ export function IndividualAdd({
       userMessage("Kullstorleken måste vara störren än 0.", "warning");
       return false;
     }
+    if (individual.litter_size6w < 1) {
```

Comment:
It good be good to make sure that litter_size6w doesn't become too big, like
```
if (individual.litter_size6w > individual.litter_size) {
userMessage("Kullstorleken efter 6 veckor får inte vara större än kullstorleken vid födseln.", "warning");
      return false;
    }
```

Reply:
Very good suggestion